test/integration/CMakeLists.txt

```diff
@@ -11,7 +11,11 @@ file(GLOB modfiles CONFIGURE_DEPENDS "${NMODL_PROJECT_SOURCE_DIR}/test/integrati
 foreach(modfile ${modfiles})
   get_filename_component(modfile_name "${modfile}" NAME)
   add_test(NAME ${modfile_name} COMMAND ${CMAKE_BINARY_DIR}/bin/nmodl ${modfile})
-  set_tests_properties(${modfile_name}
-                       PROPERTIES ENVIRONMENT PYTHONPATH=${PROJECT_BINARY_DIR}/lib:$ENV{PYTHONPATH})
+  if(WIN32)
+    set(TEST_PYTHON_PATH "${PROJECT_BINARY_DIR}/lib;$ENV{PYTHONPATH}")
```

Comment:
Since we share variable names with everyone else, I think it's common to namespace them.

Reply:
They are scoped though and shouldn't leak outside `test/integration`.

Reply:
We're willing to find out exactly how the scoping breaks down, just to avoid having a prefix `NMODL_`?

Reply:
Done in [4c36b78](https://github.com/BlueBrain/nmodl/pull/1354/commits/4c36b788631ac0935b042b7730143f73e1c9305d)